components/insight/SRC/org/openmicroscopy/shoola/agents/fsimporter/chooser/FileSelectionTable.java

```diff
@@ -358,7 +358,7 @@ private void removeSelectedFiles()
 	    int[] rows = table.getSelectedRows();
 	    if (rows == null || rows.length == 0) return;
 	    DefaultTableModel dtm = (DefaultTableModel) table.getModel();
-	    Vector<?> v = dtm.getDataVector();
+	    Vector v = dtm.getDataVector();
```

Comment:
Out of curiosity, what goes wrong here? This will probably create a yellow warning in Eclipse; is there any other solution?